ios/brave-ios/Sources/Brave/WebFilters/DownloadResourceInterface.swift

```diff
@@ -66,11 +57,17 @@ extension DownloadResourceInterface {
   ///
   /// - Note: Returns nil if the cache folder does not exist
   var createdCacheFolderURL: URL? {
-    guard let folderURL = Self.cacheFolderDirectory.url else { return nil }
-    let cacheFolderURL = folderURL.appendingPathComponent(cacheFolderName)
+    guard
+      let folderURL = try? AsyncFileManager.default.url(
+        for: .cachesDirectory,
+        in: .userDomainMask
+      ).appending(path: cacheFolderName)
+    else {
+      return nil
+    }
```

Comment:
Again, not detached but I can handle this if you would like.

Reply:
Same as above